openslides/users/views.py

```diff
@@ -39,8 +40,10 @@ def check_view_permissions(self):
         """
         if self.action in ('list', 'retrieve'):
             result = self.get_access_permissions().check_permissions(self.request.user)
-        elif self.action in ('metadata', 'update', 'partial_update'):
+        elif self.action == 'metadata':
             result = has_perm(self.request.user, 'users.can_see_name')
+        elif self.action in ('update', 'partial_update'):
+            result = self.request.user.is_authenticated()
```

Comment:
Ok but now we need to check the has_perm(self.request.user, 'users.can_see_name') inside the update method.